src/App.js

```diff
@@ -44,9 +38,16 @@ const notifyUser = announcement => {
 }
 
 const PageRoute = ({ path, children }) => {
+  const [livesiteDoc, setLivesiteDoc] = useState(false)
+
+  useEffect(() => {
+    const unsubscribe = getLivesiteDoc(setLivesiteDoc)
+    return unsubscribe
+  }, [setLivesiteDoc])
```

Comment:
I'm not too familiar with the `useEffect` hook, but does this change on `setLivesiteDoc` or should it change on `livesiteDoc` changing?

Reply:
you're right, it's not supposed to be there. It's only supposed to fetch the `LivesiteDoc` on the initial render.